src/ui.rs

```diff
@@ -3687,43 +3687,40 @@ impl CodeWindow {
             return;
         }
         ui.should_redraw = true;
-        let mut breakpoint_id = None;
-        for (id, breakpoint) in debugger.breakpoints.iter() {
-            match &breakpoint.on {
-                BreakpointOn::Line(bp) if bp.path == tab.path_in_symbols && (bp.line == tab.area_state.cursor + 1 || bp.adjusted_line == Some(tab.area_state.cursor + 1)) => {
-                    if delete {
-                        debugger.remove_breakpoint(id);
-                        return;
-                    }
-                    breakpoint_id = Some(id.clone());
-                    if !edit_condition {
-                        let enable = !breakpoint.enabled;
-                        let r = debugger.set_breakpoint_enabled(id, enable);
-                        report_result(state, &r);
-                    }
-                    break;
+        let lb = LineBreakpoint {path: tab.path_in_symbols.clone(), file_version: FileVersionInfo::default(), line: tab.area_state.cursor + 1, adjusted_line: None};
```

Comment:
`Self::make_breakpoint_for_current_line(tab)`, and use `lb` below instead of the existing call to `Self::make_breakpoint_for_current_line(tab)`.